lib/matplotlib/tests/test_tightlayout.py

```diff
@@ -19,7 +19,7 @@ def example_plot(ax, fontsize=12):
     ax.set_title('Title', fontsize=fontsize)
 
 
-@image_comparison(['tight_layout1'])
+@image_comparison(['tight_layout1'], tol=1.87)
```

Comment:
Do you need to increase the tolerance, or can you not set the rcParam like the other test?

Reply:
No, its really strange - the whole figure is the same *except* for the blue line.

Reply:
This was the macOS failure with new Ghostscript? So this can now be rebased out.

Reply:
I don't think so, but maybe...